app.ts

```diff
@@ -0,0 +1,462 @@
+  })
+    .then((res) => res.text())
+    .then((data) => {
+      data = JSON.parse(data);
+      let count = data;
+      let currentStats =
+        "Showing results from " +
+        paramOne +
+        " to " +
+        paramTwo +
+        " out of " +
+        count +
+        " results.";
+      pageStats.innerHTML = currentStats;
+    });
+}
+
+// Loads all intial fields/data
+
+window.onload = function () {
+  let windowHeight = window.innerHeight;
+
+  if (windowHeight < 600 && windowHeight >= 480) {
+    paramOne = paramOne;
+    paramTwo = parseInt(paramOne) + 6;
+    paramTwo = paramTwo.toString();
+    stats();
+    getColumns();
+    getTable();
+  } else if (windowHeight < 480 && windowHeight >= 400) {
+    paramOne = paramOne;
+    paramTwo = parseInt(paramOne) + 4;
+    paramTwo = paramTwo.toString();
+    stats();
+    getColumns();
+    getTable();
+  } else if (windowHeight < 400 && windowHeight > 300) {
+    paramOne = paramOne;
+    paramTwo = parseInt(paramOne) + 2;
+    paramTwo = paramTwo.toString();
+    stats();
+    getColumns();
+    getTable();
+  } else if (windowHeight <= 300) {
+    paramOne = paramOne;
+    paramTwo = parseInt(paramOne) + 1;
+    paramTwo = paramTwo.toString();
+    stats();
+    getColumns();
+    getTable();
+  } else {
+    stats();
+    getColumns();
+    getTable();
+  }
+};
+
+// Displaying data into html
+
+//// Targets heading div to creating the 1st row(Column names)
+
+function colHeading(heading: string) {
+  let headingCol = `<div id="col_heading" class="col_heading">${heading}</div>`;
+  headings.innerHTML += headingCol;
+}
+
+//// Targets content div to create the actual table and fill with data
+
+function cols(content: any) {
+  let rows = `<div id=row-${content[0]} class="rows"></div>`;
+
+  content_cols.innerHTML += rows;
+
+  let finalRow: any = document.querySelector("#row-" + content[0] + ".rows");
+
+  for (let x = 0; x < content.length; x++) {
+    let rowCols = `<div class="row-cols">${content[x]}</div>`;
+    finalRow.innerHTML += rowCols;
+  }
+}
+
+// Clear Table (content div)
+
+function clearTable() {
+  content_cols.innerHTML = clear;
+}
+
+// Jump to ID
+
+function idJump() {
+  let windowHeight = window.innerHeight;
+  let toConvert: any;
+
+  if (windowHeight < 600 && windowHeight >= 480) {
+    toConvert = 6;
+  } else if (windowHeight < 480 && windowHeight >= 400) {
+    toConvert = 4;
+  } else if (windowHeight < 400 && windowHeight > 300) {
+    toConvert = 2;
+  } else if (windowHeight <= 300) {
+    toConvert = 1;
+  } else {
+    toConvert = 9;
+  }
+
+  let currentOne: any = paramOne;
+  let currentTwo: any = paramTwo;
+  let search: any = document.querySelector("#idJump");
+  let searchOne = search.value;
+  let convert = parseInt(searchOne) + toConvert;
+  let searchTwo = convert.toString();
+  paramOne = searchOne;
+  paramTwo = searchTwo;
+
+  if (
+    searchOne !== "" &&
+    searchOne <= 999990 &&
+    searchOne >= 0 &&
+    searchOne !== "e"
```

Comment:
I think the standard way in IMQS is to put the if statement's condition on the same line. Follow up with Fritz on this.